eng/pipelines/templates/variables/image.yml

```diff
@@ -2,18 +2,18 @@
 
 variables:
   - name: LINUXPOOL
-    value: azsdk-pool-mms-ubuntu-2004-general
+    value: azsdk-pool
   - name: WINDOWSPOOL
-    value: azsdk-pool-mms-win-2022-general
+    value: azsdk-pool
   - name: MACPOOL
     value: Azure Pipelines
 
   - name: LINUXVMIMAGE
-    value: azsdk-pool-mms-ubuntu-2004-1espt
+    value: ubuntu-22.04
```

Comment:
Any place where this is used outside of the 1ESPT context I think we also need to add an image demand to get the correct image (I think 1ESPT does this for you).

Reply:
From a quick [scan](https://github.com/search?q=repo%3AAzure%2Fazure-sdk-for-python%20LINUXVMIMAGE&type=code) I only noticed one case for sure https://github.com/Azure/azure-sdk-for-python/blob/f20fdb40305c756b00b12859e16e83eec5859fe7/eng/common/pipelines/templates/jobs/prepare-pipelines.yml#L17

Reply:
@mikeharder  explicitly told me to use the shortnames because he added the mapping in the pool. @mikeharder do you want to chime in here?

Reply:
I'll just swap over to the full names.

Reply:
I'm pretty sure every pipeline in specs repo is working correctly with the new pool, if these examples are helpful:

https://github.com/search?q=repo%3AAzure%2Fazure-rest-api-specs%20ubuntu-22.04&type=code

Reply:
That really does help I can just copy/pasta. Cheers @mikeharder

Reply:
Eh. I don't want to change common for this. Just updating to use the long names.

Reply:
@weshaggard due to lack of long names in the pool, it actually just figures it out. [evidence](https://dev.azure.com/azure-sdk/internal/_build/results?buildId=4724209&view=logs&j=011e1ec8-6569-5e69-4f06-baf193d1351e&t=669fbf8e-0dd1-4c6c-afde-21f998fb9bd7) 

I think this is ready to just go in.